torch/distributed/optim/functional_adadelta.py

```diff
@@ -22,6 +22,7 @@ def __init__(
         rho: float = 0.9,
         eps: float = 1e-6,
         weight_decay: float = 0.0,
+        foreach: bool = False,
```

Comment:
cc @mrshenli is this ok? Do you want this flexibility on the distributed side or are you always passing a single parameter and want to keep this as False?